code-pushup.preset.ts

```diff
@@ -0,0 +1,162 @@
+    refs: [
+      {
+        type: 'group',
+        plugin: 'js-packages',
+        slug: 'npm-audit',
+        weight: 1,
+      },
+    ],
+  },
+  {
+    slug: 'updates',
+    title: 'Updates',
+    description: 'Finds **outdated** 3rd-party packages.',
+    refs: [
+      {
+        type: 'group',
+        plugin: 'js-packages',
+        slug: 'npm-outdated',
+        weight: 1,
+      },
+    ],
+  },
+];
+
+export const lighthouseCategories: CategoryConfig[] = [
+  {
+    slug: 'performance',
+    title: 'Performance',
+    refs: [lighthouseGroupRef('performance')],
+  },
+  {
+    slug: 'a11y',
+    title: 'Accessibility',
+    refs: [lighthouseGroupRef('accessibility')],
+  },
+  {
+    slug: 'best-practices',
+    title: 'Best Practices',
+    refs: [lighthouseGroupRef('best-practices')],
+  },
+  {
+    slug: 'seo',
+    title: 'SEO',
+    refs: [lighthouseGroupRef('seo')],
+  },
+];
+
+export const eslintCategories: CategoryConfig[] = [
+  {
+    slug: 'bug-prevention',
+    title: 'Bug prevention',
+    description: 'Lint rules that find **potential bugs** in your code.',
+    refs: [{ type: 'group', plugin: 'eslint', slug: 'problems', weight: 1 }],
+  },
+  {
+    slug: 'code-style',
+    title: 'Code style',
+    description:
+      'Lint rules that promote **good practices** and consistency in your code.',
+    refs: [{ type: 'group', plugin: 'eslint', slug: 'suggestions', weight: 1 }],
+  },
+];
+
+export const coverageCategories: CategoryConfig[] = [
+  {
+    slug: 'code-coverage',
+    title: 'Code coverage',
+    description: 'Measures how much of your code is **covered by tests**.',
+    refs: [
+      {
+        type: 'group',
+        plugin: 'coverage',
+        slug: 'coverage',
+        weight: 1,
+      },
+    ],
+  },
+];
+
+export const jsPackagesCoreConfig = async (): Promise<CoreConfig> => {
+  return {
+    plugins: [await jsPackagesPlugin({ packageManager: 'npm' })],
+    categories: jsPackagesCategories,
+  };
+};
+
+export const lighthouseCoreConfig = async (
+  url: string,
+): Promise<CoreConfig> => {
+  return {
+    plugins: [
+      await lighthousePlugin(url, {
+        chromeFlags: DEFAULT_FLAGS.concat(['--headless']),
+      }),
+    ],
+    categories: lighthouseCategories,
+  };
+};
+
+export const eslintCoreConfigNx = async (
+  projectName?: string,
+): Promise<CoreConfig> => {
+  return {
+    plugins: [
+      await eslintPlugin(
+        await (projectName
+          ? eslintConfigFromNxProject(projectName)
+          : eslintConfigFromAllNxProjects()),
+      ),
+    ],
+    categories: eslintCategories,
+  };
+};
+
+export const coverageCoreConfigNx = async (
+  projectName?: string,
+): Promise<CoreConfig> => {
+  const targetNames = ['unit-test', 'integration-test'];
+  const targetArgs = [
+    '-t',
+    'unit-test',
+    'integration-test',
+    '--coverage.enabled',
+    '--skipNxCache',
+  ];
+  const rootArgs = ['nx', 'run-many'];
```

Comment:
Unused variable:

```suggestion
```

Reply:
removed

Reply:
You haven't pushed anything since my last review :sweat_smile: